test/helm/Dockerfile

```diff
@@ -66,6 +68,23 @@ SHELL [ "/bin/sh", "-c" ]
 
 RUN helm version | grep "3.7.0"
 
+#--------------------------------------
+# install helm user 1000:1000
+#--------------------------------------
+FROM build as testd
+
+USER 1000:1000
```

Comment:
This should not work and we should not support it.

Either we should install as `1000:0` and execute as `1000:1000`

Reply:
sure, was for testing only. will revert. for fast fix we should make the new paths reable and executable by others. that should hopefully fix at leat running the full renovate image with `1000:1000`?

@rarkins WDYT? Should we support running renovate image as `1000:1000`? That won't support `binarySource=install` as that requires root group to write to `/opt/buildpack`. I don't like to make all those folders writable by all.

Reply:
Yes, `775` for non-root install and `755` for root install should be fine.
So the change you made on the `setup_directories` is fine.
Additionally you need to change the usmask here: https://github.com/containerbase/buildpack/blob/main/src/usr/local/buildpack/utils/constants.sh#L15-L17

And adapt the tests :)